src/sentry/features/__init__.py

```diff
@@ -63,6 +63,7 @@
 default_manager.add("auth:register", SystemFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:create", SystemFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("relocation:enabled", SystemFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("hybrid-cloud:region-provisioning-allow-list", SystemFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
Can this be merged before the flagr feature exists?

Reply:
Yes, feature flags can be defined here before they are added to flagr.